Doc/whatsnew/3.11.rst

```diff
@@ -377,6 +377,10 @@ Other Language Changes
   coerces negative zero to zero after rounding to the format precision.  See
   :pep:`682` for more details.  (Contributed by John Belmonte in :gh:`90153`.)
 
+* :pep:`563` Postponed Evaluation of Annotations, ``__future__.annotations`` has been indefinitely postponed.
```

Comment:
I don't think this fits here as there's no change.

Reply:
Personally I rushed to the release page to see the status of pip 563 as in 3.10 it was marked as being released in 3.11. And was surprised to see that there was no mention of it, despite the fact that it has been pulled from 3.11.

I hadn't seen any of the information regarding it's changed status, it would be of value to be to have it in here. Maybe it could be moved to the full-form changelog?

Reply:
Perhaps in the "Improved Modules" section it could be listed under `__future__`?

Reply:
I would suggest something along the lines of a "Non-futures Related to Type Hints" subsection of the "New Features Related to Type Hints" section above.  Maybe not that particular name, but I enjoyed coming up with it :)

Lines in our reST docs should be wrapped at 80 columns as [noted in the devguide](https://devguide.python.org/documenting/#use-of-whitespace).

Reply:
Done, I used your suggested name because I didn't understand what you meant by it😁